Description: autotest: disable lcores_autotest on ppc
 This test consistently times out on ppc64 builds. Disable it.
Author: Luca Boccassi <bluca@debian.org>
Forwarded: yes, https://patchwork.dpdk.org/project/dpdk/patch/20210420114508.397249-1-luca.boccassi@gmail.com/
--- a/app/test/meson.build
+++ b/app/test/meson.build
@@ -213,7 +213,6 @@
         ['hash_autotest', true],
         ['interrupt_autotest', true],
         ['ipfrag_autotest', false],
-        ['lcores_autotest', true],
         ['logs_autotest', true],
         ['lpm_autotest', true],
         ['lpm6_autotest', true],
@@ -420,6 +419,11 @@
 	test_deps += 'crypto_scheduler'
 endif
 
+# This test consistently times out on ppc64
+if arch_subdir != 'ppc'
+	fast_tests += [['lcores_autotest', true]]
+endif
+
 foreach d:test_deps
 	def_lib = get_option('default_library')
 	test_dep_objs += get_variable(def_lib + '_rte_' + d)
